py/sight/service_utils.py

```diff
@@ -105,13 +105,7 @@ def get_service_id() -> str:
 
 
 def get_port_number() -> str:
-  if 'PORT' in os.environ:
-    return os.environ['PORT']
-  # need to use secure channel for cloud run server
-  elif (FLAGS.deployment_mode in ['local', 'vm']):
-    return '8080'
-  else:
-    return FLAGS.port
+    return os.environ.get('PORT', FLAGS.port)
```

Comment:
this will impact the case when we need to hit the local server which require port 8080

Reply:
The problem I was getting was in vm mode where I wanted to override this value but couldn't. Maybe its good enough to just default local to 8080 and then allow vm to use the flag or the environment?